skyportal/services/test_server/test_server.py

```diff
@@ -210,9 +235,35 @@ def make_app():
         app = make_app()
         server = tornado.httpserver.HTTPServer(app)
         port = cfg["test_server.port"]
+        smtp_port = cfg["test_server.smtp_port"]
         server.listen(port)
 
         refresh_cache_days = cfg["test_server.refresh_cache_days"]
 
+        log(f"Listening for test SMTP requests on port {smtp_port}")
+        # SMTP TLS context
+        context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+        context.load_cert_chain(
+            "skyportal/services/test_server/cert.pem",
```

Comment:
May want to specify this as a path relative to `__file__`

Reply:
Sure, we can do that